packages/pwa-kit-dev/scripts/upload.test.js

```diff
@@ -11,21 +11,35 @@ const Utils = require('./utils')
 
 const uploadBundle = require('./upload.js')
 
-test('uploadBundle fails with no options, no project slug, or empty project slug', () => {
-    ;[undefined, {}, {projectSlug: ''}].forEach((options) => {
-        Utils.fail.mockClear()
-        Utils.fail.mockImplementationOnce(() => {
-            throw new Error()
-        })
+let realFail
+beforeEach(() => {
+    realFail = Utils.fail
+    Utils.fail.mockImplementation((errorMessage) => {
+        throw new Error(errorMessage)
+    })
+})
```

Comment:
At one point a test in this file was failing because we added a new required option to `uploadBundle` (`origin`), and it was tripping me up because the option validation has no effect unless `Utils.fail` throws an error.